### ISSUE TITLE
PropS Wrapper missing propositions

### ISSUE THREAD

**Issue by gabrielStanovsky** [vered1986/OKR#14]
Collected by @kleinay.
There is a bug in the props-wrapper. The json props-wrapper returned:
```
{'Entities': {'A1': ("Boy Scouts ' ` perversion ' files set to be released",
   (0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10))},
 'Predicates': {'P1': {'Arguments': ['A1'],
   'Bare predicate': ('travel', (11,)),
   'Head': {'Lemma': 'travel', 'POS': 'VBP', 'Surface': ('travel', [11])},
   'Template': '{A1} travel'},
  'P2': {'Arguments': ['P3'],
   'Bare predicate': ('set', (7,)),
   'Head': {'Lemma': 'set', 'POS': 'VBN', 'Surface': ('set', [7])},
   'Template': 'set {P3}'}},
 'Sentence': "Boy Scouts ' ` perversion ' files set to be released travel"}
```
Note ```P3``` is an argument in ```P2``` but no proposition ```P3``` is being declared.

**Comment by gabrielStanovsky** [comment]
@kleinay, is this a proper sentence from the tweets:
```
Boy Scouts ' ` perversion ' files set to be released travel
```
Seems partial?

**Comment by gabrielStanovsky** [comment]
I think I fixed this bug in #15, take a look.
It works on this sentence (added to raw_sentences).

If we're satisfied, we can merge that PR, and close this issue.

**Comment by kleinay** [comment]
Accepted it. You can merge

בתאריך יום ב׳, 14 באוג׳ 2017 ב-18:57 מאת Gabriel Stanovsky <
notifications@github.com>:

**Comment by gabrielStanovsky** [comment]
I think I'll try to fix #12 in the same PR, so waiting a little bit with the merge.

**Comment by gabrielStanovsky** [comment]
Fixed, closing issues.
